saleor/graphql/schema.graphql

```diff
@@ -2632,7 +2637,7 @@ type Mutation {
   shippingPriceDelete(id: ID!): ShippingPriceDelete
   shippingPriceBulkDelete(ids: [ID]!): ShippingPriceBulkDelete
   shippingPriceUpdate(id: ID!, input: ShippingPriceInput!): ShippingPriceUpdate
-  shippingPriceTranslate(id: ID!, input: NameTranslationInput!, languageCode: LanguageCodeEnum!): ShippingPriceTranslate
+  shippingPriceTranslate(id: ID!, input: DescriptionTranslationInput!, languageCode: LanguageCodeEnum!): ShippingPriceTranslate
```

Comment:
`DescriptionTranslationInput` name does not make sense, I suggest `ShippingPriceTranslationInput`